test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests.xml

```diff
@@ -59,5 +64,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
         <output type="success" caseSensitive="yes" regex="no">$NOREADIPINFOFORRAS_MESSAGE$</output>
         <output type="failure" caseSensitive="yes" regex="no">$READIPINFOFORRAS_MESSAGE$</output>
     </test>
+
+    <test id="test -XX:OnOutOfMemoryError=">
+        <command>$EXE$ $ONOUTOFMEMORYERROR_EQUALS$"java -version" $ONOUTOFMEMORYERROR_JAR$</command>
+        <output type="success" caseSensitive="yes" regex="no">$JAVALANGOUTOFMEMORYERROR$</output>
+        <output type="success" caseSensitive="yes" regex="no">$JVMREQUESTINGTOOLDUMP$</output>
```

Comment:
A cmdlinetester test passes if any of the success conditions occur, but in this case we only want the test to pass if all conditions occur. Use "success" for only one condition and "required" for the others.